dt-core/admin/menu/tabs/tab-featured-extensions.php

```diff
@@ -92,6 +104,10 @@ public function box_message() {
             //install plugin
             $this->install_plugin( sanitize_text_field( wp_unslash( $_POST["install"] ) ) );
             exit;
+        } elseif ( isset( $_POST["deactivate"] ) && is_admin() && isset( $_POST["_ajax_nonce"] ) && check_ajax_referer('portal-nonce', sanitize_key( $_POST["_ajax_nonce"] ) ) && current_user_can( "manage_dt" ) ) {
```

Comment:
You are missing a space after the opening parenthesis in the check_ajax_referer function that is causing Travis to fail.